index.html

```diff
@@ -54,5 +54,195 @@ <h6>LET'S CONNECT</h6>
+              <a href="" class="role"><h6 class="roletext">Back End Dev</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="year"><h6 class="yeartext">2015</h6></a>
+            </div>
+          </li>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
+          <li class="btn">
+            <a><h6>See Project</h6></a>
+          </li>
+        </ul>
+      </article>
+      <article class="card">
+        <a href="" class="snapshot">
+          <img src="/Worksection Pics/snap4.png" alt="" />
+        </a>
+        <ul class="leftblock2">
+          <li .primarytext>
+            <h2>Multi-Post Stories</h2>
+            <div class="frame">
+              <a href="" class="client"><h6 class="clienttext">Canopy</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="role"><h6 class="roletext">Back End Dev</h6></a>
+              <a href=""><img src="/Worksection Pics/Counter.png" alt="" /></a>
+              <a href="" class="year"><h6 class="yeartext">2015</h6></a>
+            </div>
+          </li>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+
+          <li class="tags">
+            <a href="" class="tag1"><h6>html</h6> </a
+            ><a href="" class="tag2"><h6>css</h6> </a
+            ><a href="" class="tag3"><h6>javascript</h6> </a>
+          </li>
+          <li class="btn">
+            <a><h6>See Project</h6></a>
+          </li>
+        </ul>
+      </article>
+    </section>
+    <section class="about">
+      <h2>
+        About <br />
+        Myself
+      </h2>
+
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+
+      <h6>LET'S CONNECT</h6>
+
+      <ul class="social-container">
+        <li>
+          <a href=""><img src="/Pictures/Social/github.png" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="/Pictures/Social/linkedin.png" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="/Pictures/Social/victory.png" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="/Pictures/Social/twitter.png" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="/Pictures/Social/m.png" alt="" /></a>
+        </li>
+      </ul>
+      <a class="btn1">Get my resume</a>
```

Comment:
- According to the project requirements, the button should follow the button styling: Active, Hover, and Pressed. see the design below, Kindly make sure you change the hover color to purple or blue  your is gray .stick to the Figma design you chose as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). You can check out the Figma design specification for the exact properties ( color )used👍🏽
 : 
 
## The hover in Figma design 

![image](https://user-images.githubusercontent.com/95297251/192114880-d813d365-1e3e-4c96-8e5a-7f1a74543c99.png)

## Your design 

![image](https://user-images.githubusercontent.com/95297251/192114934-4b9d7fab-5708-4de1-a0fc-bd9b470ec673.png)

Reply:
Fixed